tests/bricks/test_conv.py

```diff
@@ -259,14 +229,14 @@ def test_convolutional_sequence():
     batch_size = 5
     activation = Rectifier().apply
 
-    conv = ConvolutionalLayer(activation, (3, 3), 5,
-                              (pooling_size, pooling_size),
-                              weights_init=Constant(1.),
-                              biases_init=Constant(5.))
+    conv = ConvolutionalActivation(activation, (3, 3), 5,
```

Comment:
A `ConvolutionalActivation` has always been a `Convolutional` and and `Activation` sandwiched together; therefore the `Convolutional` has parameters that need to be set. This was not my doing or my preference. (In fact, I'd be in favour of just allowing `Activation`s to be put inside of `ConvolutionalSequence`s, since the input-to-output shape inference is trivial, and removing this class too).